### ISSUE TITLE
This file type is not supported error after prefill

### ISSUE THREAD

**Issue by Lorempn2** [alessiomaffeis/vue-picture-input#42]
Hi,

The plugin works great without prefill. However, if I prefill an image that works fine. But if I then want to upload/select a new image, I get the error (This file type is not supported). If I dismiss the error the image is still inserted, and it is a valid image. If I dont prefill any image and select the same image, I do not get the error. 

I use: accept="image/jpeg,image/png"

**Comment by alessiomaffeis** [comment]
Uhm, I can’t reproduce the issue. Would you mind attaching the pictures and a snippet of the code where the component is in use?

**Comment by Lorempn2** [comment]
Sure.

 <picture-input ref="logoInput" :prefill="logo_path"  accept="image/jpeg,image/png"  @change="onLogoSelected" width="300" height="90"  size="10" buttonClass="btn btn-sm white" :customStrings="{drag:'Select an image'}"></picture-input>

`logo_path` is a full URL pointing to an online image (a jpg).

The input is prefilled just fine, however if I then want to upload another image, I select the image from the file browser, I get the message `This file type is not supported` and then the image is still inserted. 

If I remove the prefill or the `accept` prop its working fine.

**Comment by alessiomaffeis** [comment]
Thanks, I will look into it! ;)

**Comment by francislavoie** [comment]
I'm seeing this same issue, prefill using a data URI image.

Minimal example:

```
            <picture-input 
                ref="pictureInput"
                width="160"
                height="100"
                margin="16"
                accept="image/jpeg"
                size="10"
                button-class="btn"
                :prefill="clientImage"
                :custom-strings="{
                    upload: '<h1>Bummer!</h1>',
                    drag: 'Drag a 😺 GIF or GTFO'
                }"
                >
            </picture-input>
```

Where `clientImage` is a string starting with `data:image/jpeg;base64,`

**Comment by Toomaaa** [comment]
Hi,

You have to accept `image/2q==` in order to fix that behavior.

i.e
`accept="image/jpeg,image/png,image/2q=="`

**Comment by francislavoie** [comment]
Oh wow, it's looking at the end of the string for the type? Interesting find!

**Comment by Toomaaa** [comment]
@francislavoie Apparently yes, because I've just tested with another image in .JPG and now the image type is `image/z`
And the image finishes by `...KUkkkkp//Z`

**Comment by Toomaaa** [comment]
Yes after some tests, it seems that it could be different depending on the pictures... So not sure that you can't make a rule (I've tested several pictures and I saw 6-7 types minimum)

**Comment by francislavoie** [comment]
Yeah it won't work as-is, need a code change to handle strings starting with `data:` and grab the image type from that if possible.

**Comment by mikebz** [comment]
I am having a similar issue.  I am trying to support users going back and forth on the app and no reloading the image they have uploaded.  Preferably I'd like to prefill the picture if the user goes back from Vuex store.  However when I do a prefill option I get an error message on the normal upload:

```
                    <picture-input 
                        ref="pictureInput"
                        name="pictureInput"
                        accept="image/jpeg,image/png"
                        height="400"
                        width="600"
                        size="10" 
                        button-class="btn btn-secondary"
                        @change="onPictureChange"
                        :prefill="imageUrl()"
                        :prefillOptions="{mediaType: 'image/jpeg,image/png'}"
                        >
```

functions:
```
            onPictureChange(image) {
                log.debug("picture changed")
                if(image) {
                    this.uploadedFile = image
                }
            },
            imageUrl() {
                return this.uploadedFile
            },
```

**Comment by abdulmalikgh** [comment]
I am also having the same issue. I am using image from an api endpoint but when I insert the image , i get this error " Failed loading prefill image: TypeError: Failed to fetch "
ref="picture"
                                @remove="onRemoved"
                                @change="uploadImage"
                                :prefill="picture"
                                :prefillOptions="{mediaType: 'image/jpeg'}"
                                :width="300"
                                :removable="true"
                                removeButtonClass="btn btn-danger btn-sm"
                                :height="250"
                                accept="image/jpeg, image/png, image/gif"
                                buttonClass="btn btn-success btn-sm"
                                :customStrings="{
                                upload: '<h1>Upload it!</h1>',
                                drag: 'Drag and drop your image here'}">
                            </picture-input>
picture: https://acabeststaging.s3.amazonaws.com/images/quizpack-thumbnails/7bu8Kf9XV0VhrfG5ADXz6WxIorzKbW5YTGoS2I6M.jpeg

**Comment by josoye24** [comment]
Am having the same issue any one with working solution.
Please help!

**Comment by alessiomaffeis** [comment]
This should be fixed in 3.0.0
